src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -390,12 +390,12 @@
     <Compile Include="Implementation\GoToDefinition\GoToDefinitionCommandHandler.cs" />
     <Compile Include="Implementation\GoToDefinition\GoToDefinitionHelpers.cs" />
     <Compile Include="Implementation\GoToDefinition\IGoToDefinitionService.cs" />
-    <Compile Include="Implementation\Highlighting\AbstractKeywordHighlighter.cs" />
-    <Compile Include="Implementation\Highlighting\HighlighterTagProducer.cs" />
-    <Compile Include="Implementation\Highlighting\HighlighterViewTaggerProvider.cs" />
-    <Compile Include="Implementation\Highlighting\HighlightingService.cs" />
-    <Compile Include="Implementation\Highlighting\HighlightingTagSource.cs" />
-    <Compile Include="Implementation\Highlighting\HighlightTag.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\AbstractKeywordHighlighter.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\HighlighterTagProducer.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\HighlighterViewTaggerProvider.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\HighlightingService.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\HighlightingTagSource.cs" />
+    <Compile Include="Implementation\KeywordHighlighting\HighlightTag.cs" />
```

Comment:
I would have liked to rename all files, namespaces and types appropriately, but the changes are spread all over and have to update all tests. In the end, settled for a cheaper alternative to just rename this folder :wink: